framework/src/org/checkerframework/framework/qual/AnnotatedFor.java

```diff
@@ -32,7 +35,8 @@
  *
  * @checker_framework.manual #compiling-libraries Compiling partially-annotated libraries
  */
-@Target({TYPE, METHOD, CONSTRUCTOR}) // permitting on PACKAGE would be too error-prone
+@Target({TYPE, METHOD, CONSTRUCTOR, PACKAGE})
```

Comment:
We decided that there was not reason not to allow this.  This allows an entire package to be marked annotated, which could be useful.